hack/e2e.sh

```diff
@@ -23,11 +23,11 @@ echo "IMAGE=${IMAGE}"
 echo "IMAGE_TAG=${IMAGE_TAG}"
 
 # Use defined PULL_SECRET or fall back to CI location
-PULL_SECRET=${PULL_SECRET:-/tmp/cluster/pull-secret}
+PULL_SECRET=${PULL_SECRET:-/var/run/secrets/ci.openshift.io/cluster-profile/pull-secret}
```

Comment:
out of curiosity, where's this documented?

Reply:
[ipi-install-install step](https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/reference/ipi-install-install) mentions pull secret location at line 15 (not sure if we have access to `CLUSTER_PROFILE_DIR` in this step, so I used the path where it gets mounted in the test container)